application/src/main/kotlin/net/corda/v5/application/marshalling/JsonSerializedBase64Config.kt

```diff
@@ -0,0 +1,12 @@
+package net.corda.v5.application.marshalling
+
+/**
+ * When objects are serialized to Json using custom serializers and they require to be encoded into base64 encoded, this
+ * enum can be used to configure how that base64 is written into the Json.
+ */
```

Comment:
I wouldn't mind a description of what each element means - even if that's forwarding to the underlying Jackson equivalents in `Base64Variant`.

What's the default? Feels like `MIME` is standard, but I wonder if `MODIFIED_FOR_URL` might be appropriate based on the docs at https://access.redhat.com/webassets/avalon/d/red-hat-jboss-enterprise-application-platform/7.0.0/javadocs/com/fasterxml/jackson/core/class-use/Base64Variant.html?

Reply:
I agree on the point about describing each, I will action that.

The default is described at the point of method overload, at the moment it's MIME_NO_LINEFEEDS as per stock Jackson. I'm not sure moving it here makes sense, because users who are using the overloads that don't take an enum are exactly the audience who wont come to this enum to read the docs (aside from the fact it could technically be different in different places - i.e. the enum declaration itself is not the decider of the default).